tickets.md

```diff
@@ -41,14 +41,14 @@ Before the event, you can either provide the name of a roommate that also regist
 <br>
 
 ### What if I don’t know anyone in the crafter community?
-A new group can be intimidating, but we hope you give SoCraTes a try. One of our organisers joined his first SoCraTes last year and felt that the atmosphere was very open to people that haven’t been there before. In addition, we are hoping to provide some [Conference Buddies][conference_buddies] so you can attend together.
+A new group can be intimidating, but we hope you give SoCraTes a try. The SoCraTes community prouds itself to be open and inclusive to everyone. In addition, we are hoping to provide some [Conference Buddies][conference_buddies] so you can attend together.
```

Comment:
We should say 
```suggestion
A new group can be intimidating, but we hope you give SoCraTes a try. The SoCraTes community aims to be open and inclusive to everyone. In addition, we are hoping to provide some [Conference Buddies][conference_buddies] so you can attend together.
```

Because we can not promise that everyone will feel included